inventory.py

```diff
@@ -237,6 +242,8 @@ def run_course_inventory() -> None:
         logger.info(f'Writing {num_enrollment_records} enrollment records to CSV')
         enrollment_df.to_csv(os.path.join('data', 'enrollment.csv'), index=False)
         logger.info('Wrote data to data/enrollment.csv')
+        logger.info(f"Writing {num_canvas_usage_records} canvas usage record to CSV")
+        canvas_usage_df.to_csv(os.path.join('data', 'canvas_usage_trails.csv'), index=False)
```

Comment:
I'm curious, what is the `trails` part supposed to mean?

Reply:
Removed it. I named it something random

Reply:
Done